fixtures/cannibalization_4_expected.rb

```diff
@@ -1 +1 @@
-Metadata.relative_path(File.expand_path(rerun_path))
+Metadata.relative_path File.expand_path(rerun_path)
```

Comment:
why did these parens go missing?

Reply:
They became consistent with [`nested_command_expected.rb`](https://github.com/samphippen/rubyfmt/blob/master/fixtures/nested_command_expected.rb) and [`command_call_raise_expected.rb`](https://github.com/samphippen/rubyfmt/blob/master/fixtures/command_call_raise_expected.rb). Same with the other fixtures that changed.

We could go the other way: change `nested_command_expected.rb` to use more parens and tweak `use_parens` to never use parens with `raise`.

Reply:
Alternative in 3cfae79 which is more consistent with the previous behaviour. Still changes a couple of fixtures, but all of these parens are back.